assets/app/view/chat.rb

```diff
@@ -14,6 +14,7 @@ class Chat < Snabberb::Component
 
     def render
       @connection.subscribe('/chat', 0) do |data|
+        data[:created_at] = Time.at(data[:created_at])
```

Comment:
why is this done?

Reply:
The time is stored on the server as a timestamp, but then it's handy to have the variable as a Time object that we can directly call its methods